plugins/CoreHome/Columns/VisitorDaysSinceFirst.php

```diff
@@ -13,6 +13,7 @@
 use Piwik\Tracker\Request;
 use Piwik\Tracker\Visitor;
 
+// TODO: it would be better to save the time duration since the first visit rather than just the number of days, since that cuts of information. make matomo 4 issue?
```

Comment:
Sure, be good to create an issue, and we shall see if/when to schedule it later :+1:

Reply:
Created issue https://github.com/matomo-org/matomo/issues/14094